jee-commons/src/main/java/biz/paluch/jee/commons/BeanLookup.java

```diff
@@ -136,5 +153,14 @@ public static BeanManager beanManager() {
     public static void reset() {
         lookupStrategy = new BeanManagerLookupStrategy();
     }
+    
+    private static BeanManager beanManagerLookup() {
+        for(String jndiName : BEANMANAGER_JNDI_NAMES) {
+            BeanManager beanManager = NamingLookup.doLookup(jndiName);
```

Comment:
Logging adds another dependency to the project, therefore I do not like that approach. Throwables provide `addSuppressed` since Java 7 for such cases. All other exceptions, that occur during the lookup can be added for later inspection.